src/opcodes/opcodes.cairo

```diff
@@ -308,6 +309,11 @@ pub mod Opcode {
         engine.dstack.push_int(depth);
     }
 
+    fn opcode_size(ref engine: Engine) {
+        let size = engine.dstack.pop_byte_array().len().into();
```

Comment:
I think we just want to `peek` here, so the stack element is not removed.

Reply:
Hi @b-j-roberts I have changed it to peek and I have also fixed the file conflict